src/persist-client/examples/maelstrom/txn.rs

```diff
@@ -442,7 +442,7 @@ impl Service for TransactorService {
         // Construct requested Consensus.
         let consensus = match &args.consensus_uri {
             Some(consensus_uri) => {
-                ConsensusConfig::try_from(consensus_uri)
+                ConsensusConfig::try_from(consensus_uri, 1)
```

Comment:
let's pull the PersistConfig creation up so we can use the consensus_connection_pool_max_size option here